test/data/import/old/1.6.0-single-route.json

```diff
@@ -8,6 +8,7 @@
       "documentation": "",
       "method": "get",
       "endpoint": "",
+      "randomResponse": false,
```

Comment:
Please do not modify files in `test/data/migrations`, `test/data/import/new` and `test/data/import/old`